shelley/chain-and-ledger/formal-spec/epoch.tex

```diff
@@ -1383,7 +1331,7 @@ \subsection{Reward Update Calculation}
       & ~~~~~~~\Delta t_1 = \floor*{(\fun{tau}~{pp}) \cdot \var{totalPot}} \\
```

Comment:
another thing that was affected by the incentive decision to remove the moving averages was that now `∆r` used in the total pot needs to depend on the number of blocks produced in the epoch.  At the beginning of 6.4.3 it mentions that the monetary expansion is given by `min(eta, 1)rho(T_infty - T)`.